app/models/footnote.rb

```diff
@@ -53,7 +53,7 @@ def footnote_description
                                   right_key: [:meursing_table_plan_id, :meursing_heading_number]
 
 
-  delegate :description, :formatted_description, to: :footnote_description
+  delegate :description, :formatted_description, to: :footnote_description, allow_nil: true
```

Comment:
Because I got a serialization error when accessing `/footnotes` page on admin

![screenshot from 2017-03-14 18-16-00](https://cloud.githubusercontent.com/assets/1684845/23907529/807360b4-08e2-11e7-9169-64cdd5edd8a0.png)

with `allow_nil: true` all works